openedx/core/djangoapps/edx_global_analytics/tests/test_acceptor_api_usage_helpers.py

```diff
@@ -0,0 +1,77 @@
+"""
+Tests for OLGA acceptor api usage by edX global analytics application tasks and helper functions.
+"""
+
+import logging
+import uuid
+import unittest
+
```

Comment:
It is needed. Standard library, related third party imports and local application imports need to be separated by empty line.